generic.go

```diff
@@ -47,57 +48,42 @@ func NewGenericMultiStream(resync ResyncFn, lastIDs map[string]string, cfg Confi
 }
 
 // Publish sends an event to the default topic ("").
-// The event is cached to support client resynchronization.
 func (s *GenericStream) Publish(event *Event) {
 	s.PublishTopic("", event)
 }
 
 // PublishTopic sends an event to the specified topic.
-// The event is cached to support client resynchronization.
 func (s *GenericStream) PublishTopic(topic string, event *Event) {
 	s.broker.publish(topic, event, nil)
 }
 
 // PublishBroadcast sends an event to all connected clients across all topics.
-// Broadcasted events are not cached and their IDs are removed to prevent
-// affecting the event sequence of any specific topic.
 func (s *GenericStream) PublishBroadcast(event *Event) {
 	event.ID = ""
 	s.broker.broadcast(event)
 }
 
 // Subscribe adds a subscriber to the default topic ("") and starts sending
-// events to the provided response writer. If lastEventID is provided and
-// differs from the server's last event ID, it attempts to resynchronize
-// missing events from the cache.
-// Returns ErrCacheMiss if resynchronization is needed but events are not found in cache.
```

Comment:
Why were detailed comments removed? IMO it would be good to know that a function can return `ErrCacheMiss`.